sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/flink/FlinkPhysicalIT.java

```diff
@@ -82,13 +82,14 @@ public void jsonToString() {
     builder.add("Array := SELECT jsonArray(customerid) AS obj FROM BigJoin");
     builder.add("Agg := SELECT jsonObjectAgg('key', name) AS agg FROM BigJoin GROUP BY name");
     builder.add("ToString := SELECT jsonToString(toJson('{}')) AS obj FROM BigJoin");
```

Comment:
Can you put some more data in here? Otherwise the snapshot output looks "suspcious".

Reply:
Also, please rename the method to jsonTest.